layout/options.go

```diff
@@ -8,79 +8,58 @@ import (
 	"github.com/buildpacks/imgutil"
 )
 
-type ImageOption func(*options) error
+type ImageOption func(*imgutil.ImageOptions)
 
-type options struct {
-	platform      imgutil.Platform
-	baseImage     v1.Image
-	baseImagePath string
-	prevImagePath string
-	withHistory   bool
-	createdAt     time.Time
-	mediaTypes    imgutil.MediaTypes
+func FromBaseImagePath(name string) func(*imgutil.ImageOptions) {
+	return func(o *imgutil.ImageOptions) {
+		o.BaseImageRepoName = name
+	}
+}
+
```

Comment:
We used to have some documentation for each option, I think we should keep that.